pkg/pillar/containerd/containerd.go

```diff
@@ -93,6 +99,10 @@ func init() {
 	if vault.ReadPersistType() == types.PersistZFS {
```

Comment:
If these 2 `if` statements are mutually exclusive, let's make this a `switch` statement.

Reply:
With an upcoming PR the persist type will still be ZFS but the snapshotter used will be ext4 so this whole if will be removed.

Reply:
> upcoming PR the persist type will still be ZFS

upcoming = a week away? Or upcoming = might be in a month?

Either way, why would be not do clean code?

Reply:
Its already implemented and can be submitted in PR but would expand the scope of this PR to vaultmgr and usb install path.

Reply:
Nah, if it is a week away, then it doesn't matter. I didn't want this lingering for months. We have a long history of letting "temporary" things like this live for too long. All good, then.